### PR TITLE
Completed Issue-970 by adding code and Readme.md

### PR DESCRIPTION
**Pull request by dsrao711**
# Description

- Added script for Converting PDF to Word document-
- Added REa

## Fixes: #970

Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x ] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

Input:
op_1
Output:
op

### CODE REVIEW

**Comment by dsrao711** [comment]
Completed ISSUE 970 - PDF to word converter

File: PDFtoWord/main.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from pdf2docx import Converter
+
+pdf_file = 'sample.pdf'
```
Comment (kaustubhgupta) [inline PDFtoWord/main.py:3]: take the input for pdf file path for user

File: PDFtoWord/main.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+docx_file = 'sample.docx'
```
Comment (kaustubhgupta) [inline PDFtoWord/main.py:4]: Prompt the user whether they want to define a custom file name for the doc. If they type "Y", then take the input for the filename. If "N" then the doc file name should be the same as the pdf file added (doc). For example:

input file: abc.pdf
Do you want to provide a custom name for the word file? (Y/N):
if "Y": 
  enter the file name without extension: 
elif "N":
 output file name: abc (doc).docx

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
delete the sample.docx and sample.pdf along with following changes:

File: PDFtoWord/main.py (hunk 2 above)
Comment (dsrao711) [reply PDFtoWord/main.py:4]: Okay

File: PDFtoWord/main.py (hunk 1 above)
Comment (dsrao711) [reply PDFtoWord/main.py:3]: Okay

**Review by kaustubhgupta — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]

**Review by antrikshmisri — APPROVED** [review]

**Comment by dsrao711** [comment]
okay

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Umm, you were not required to pull these changes:
image

Neither these:
image

Nor the commit above,

You have pulled a lot of not-required commits in this PR. I would suggest you to either remove all of them or create a new PR

**Pull request closed without merging**